index.html

```diff
@@ -12,8 +12,107 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
+</head>
+
+<body>  
+  <div class = "container">
+    <header>
+  <nav>
+<a href ="#">Home</a>
+<a href ="#">About</a>
+    <a href ="#">Portfolio</a>
+        <a href ="#">Contact</a>
+
+</nav>
+
+
+</header>
+
+<section class="sub">
+
+
+<img src="photo.jpg" alt="Home Page Image.">
+
+<h1>Carmen Salley - Web Dev</h1>
+
+<br>
+  <h3>Works effectively as an individual or with a team, strong communication & organizational skills, exceptionally detail oriented, skilled in
+    Information Technology/Programming/Telecommunications</h3>
+
+<div class = "line">
 
-<body>
-  <h1>My Resume</h1>
+</section> 
+
+<div  class = "one">
+<h6>Education</h6>
+
+   <ul>
+     <li>AMERICAN INTERCONTINENTAL UNIVERSITY,
+  Hoffman Estates, IL 10/05 – 11/06
+  Bachelor of Science in Information Technology
+  Major: Information Technology/Computer Programming
+  Magna cum Laude/GPA of 3.7 // 90 credits earned </li>
+  <br>
+  <li>VOORHEES COLLEGE,
+  Denmark, S.C. 08/02 – 05/05
+  Major: Computer Science // Minor: Mathematics-- Held a GPA
+  of 3.5, Honors College. transferred
+  89 credits earned</li>
+  <br>
+  <li>Western Governor’s University
+  Bachelors of Science in Computer Science
+  Anticipated Graduation Date: 10/22</li>
+  <br>
+  <li>Lambda School
+  Full-Stack Web Development & Computer Science
+  Anticipated Graduation Date: 06/20</li>
+  </ul>
```

Comment:
Try to remember to indent your code by two spaces for each pair of tags. 🙂 It helps greatly for readability.